pkgs/swift2objc/lib/src/parser/parsers/declaration_parsers/parse_initializer_declaration.dart

```diff
@@ -0,0 +1,69 @@
+import '../../../ast/_core/shared/parameter.dart';
+import '../../../ast/declarations/compounds/members/initializer_declaration.dart';
+import '../../_core/json.dart';
+import '../../_core/parsed_symbolgraph.dart';
+import '../../_core/utils.dart';
+
+InitializerDeclaration parseInitializerDeclaration(
+  Json initializerSymbolJson,
+  ParsedSymbolgraph symbolgraph,
+) {
+  return InitializerDeclaration(
+    id: parseSymbolId(initializerSymbolJson),
+    params: _parseInitializerParams(initializerSymbolJson, symbolgraph),
+    hasObjCAnnotation: symbolHasObjcAnnotation(initializerSymbolJson),
+  );
+}
+
+List<Parameter> _parseInitializerParams(
+  Json initializerSymbolJson,
+  ParsedSymbolgraph symbolgraph,
+) {
+  final fragments = initializerSymbolJson['declarationFragments'];
+
+  final externalParamKind = 'externalParam';
```

Comment:
These could be const. But actually there's not really much point pulling them out like this, since they're only used in one place.

Reply:
yeah ur right. Removed them